<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactWidget</class>
 <widget class="QWidget" name="ContactWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>75</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>75</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget d'un contact</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QLabel" name="profileImg">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <italic>true</italic>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Prénom: </string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_prenom">
         <property name="text">
          <string>prenom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Nom : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_nom">
         <property name="text">
          <string>nom</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <italic>true</italic>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Téléphone:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_tel">
         <property name="text">
          <string>tel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Email: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_mail">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>mail</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Entreprise: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_entreprise">
         <property name="text">
          <string>entreprise</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
